output/compact.go

```diff
@@ -48,7 +48,7 @@ func uploadAndSubmitCompactSnapshot(ctx context.Context, s pganalyze_collector.C
 		if collectionOpts.OutputAsJson {
 			debugCompactOutputAsJSON(logger, compressedData)
 		} else if !quiet {
-			logger.PrintInfo("Collected compact %s snapshot successfully", kind)
+			logger.PrintVerbose("Collected compact %s snapshot successfully", kind)
```

Comment:
No strong opinion, but that's mainly for the test case - I wonder if that is helpful to have? (do you have an example of the full `--test` output before and after?)

Reply:
I thought this was called normally, but it looks like it's only called in a handful of paths like `--dry-run --benchmark`. I'll revert this change.